src/auth/components/game-img/game-img.vue

```diff
@@ -0,0 +1,108 @@
+import { ThemeStore } from '../../../_common/theme/theme.store';
+<template>
+	<div id="cover-img-credits">
+		<div class="flow-col">
+			<a class="link-unstyled -title" :href="gameUrl">{{ gameTitle }}</a>
+			<a class="link-muted -dev" :href="devUrl"
+				>by <strong>{{ developerName }}</strong></a
+			>
+			<span class="text-muted -follows">{{ gameFollows }} followers</span>
+		</div>
+		<a class="-img" :href="gameUrl">
+			<app-game-thumbnail-img :game="coverGame" />
+		</a>
+	</div>
+</template>
+
+<style scoped>
+#cover-img-credits {
```

Comment:
The parent should position this. This component should just render itself, this way it can put included anywhere